tests/bootstrap.php

```diff
@@ -154,3 +154,37 @@
 }
 
 FactoryLocator::add('Table', new TableLocator());
+
+/**
+ * Enable automatic loading of plugins from plugins.php for all tests.
+ *
+ * Call this function in your application's tests/bootstrap.php to
+ * automatically load all plugins defined in config/plugins.php before
+ * each test.
+ *
+ * @param string|null $configPath The path to the config directory. Defaults to CONFIG.
+ * @return void
+ * @since 5.3.0
+ */
+function enablePluginLoadingForTests(?string $configPath = null): void
```

Comment:
Is the intention that applications would use this to load plugins? Generally applications don't load this file, we could add this to `src/TestSuite/functions.php` which applications could either autoload or require.